### ISSUE TITLE
Content file format and discussion on extensions.

### ISSUE THREAD

**Issue by DaveDavenport** [Blezzing/blezz#12]
Could we have a specification on the format used in content? So I can keep *rofi* [blezz plugin](https://gitcrate.org/rofi/rofi-blezz) up2date.

Secondly I would like to extend it with an option so it executes the command but keeps the view open.
This way I can replace my i3-modes with a blezz script.

**Comment by Blezzing** [comment]
Yes, i will look into specifying the content format, and possibly extending it to allow for a stay open after executing functionality

**Comment by DaveDavenport** [comment]
Let me know. I have some ideas for some extensions.
Thx for blezz
